tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -1260,17 +1260,11 @@ def test_max_explicit(self, xp, sp):
 
     @testing.numpy_cupy_array_equal(sp_name='sp')
     def test_argmin(self, xp, sp):
-        # TODO(takagi) Fix axis=None
-        if self.axis is None:
-            pytest.skip()
         data = self._make_data_min(xp, sp, dense=self.dense)
         return data.argmin(axis=self.axis)
```

Comment:
Could you try the following fix to avoid a type mismatch?
```suggestion
        return xp.array(data.argmin(axis=self.axis))
```

Reply:
Could you also fix other tests?